main.py

```diff
@@ -71,7 +71,7 @@ async def on_member_join(self, member):
     async def on_message(self, message):
         await self.wait_until_ready()
 
-        if message.author.bot:
+        if message.author.bot or message.channel.id in [713841306253656064, 680851820587122700]:
```

Comment:
Move the check lower, like at line 81 for example

Reply:
Just made another PR with the asked changes(#160).
and also mentioned the reason why i made a new PR rather than editing this.
Will close this now.